input/httplisten/httplisten_test.go

```diff
@@ -101,3 +101,65 @@ input:
 		assert.Equal(map[string]interface{}{"foo": "bar"}, event.Extra)
 	}
 }
+
+func Test_input_httpslisten2_module(t *testing.T) {
+	assert := assert.New(t)
+	assert.NotNil(assert)
+	require := require.New(t)
+	require.NotNil(require)
+
+	ctx := context.Background()
+	conf, err := config.LoadFromYAML([]byte(strings.TrimSpace(`
+debugch: true
+input:
+  - type: httplisten
+    address: "127.0.0.1:8999"
+    path: "/tls2/"
+    cert: "./server.pem"
+    key:  "./server.key"
+    ca:   "./root.pem"
+        `)))
+
+	require.NoError(err)
+	require.NoError(conf.Start(ctx))
+	time.Sleep(500 * time.Millisecond)
+
+	rootPEM, err := ioutil.ReadFile("./root.pem")
+	require.NoError(err)
+	roots := x509.NewCertPool()
+	assert.NotNil(roots)
+	ok := roots.AppendCertsFromPEM(rootPEM)
+	assert.Equal(ok, true)
+
+	clientCert, err := tls.LoadX509KeyPair("./client.pem", "./client.key")
+	require.NoError(err)
+
+	// case 1: without client cert
+	tlsConfig := tls.Config{
+		RootCAs:      roots,
```

Comment:
Error: found unformatted go source: "/home/travis/gopath/src/github.com/tsaikd/gogstash/input/httplisten/httplisten_test.go"

Reply:
hmm, yeah, it needs reformatted